### PR TITLE
There will be 404 error when getting the tool, I fixed it.

### PR DESCRIPTION
**Pull request by ghost**
There will be 404 error when getting the tool, I fixed it.

### CODE REVIEW

**Comment by ghost** [comment]
PS: This is not what I did, It is from another repository.

**Review by ProximaB — APPROVED** [review]
Thanks for PR 👍

File: gettools.py [hunk 1]
```diff
@@ -71,89 +72,154 @@ def convertpath(path):
...
+	
+	urlpost15 = url + lastVersion + '/packages/com.vmware.fusion.tools.darwin.zip.tar'
+	urlpre15 = url + lastVersion + '/packages/com.vmware.fusion.tools.darwinPre15.zip.tar'
+	parser.clean()
+
+	# Download the darwin.iso tgz file
+	print('Retrieving Darwin tools from: ' + urlpost15)
+	try:
+		# Try to get tools from packages folder
+		urlretrieve(urlpost15, convertpath(dest + '/tools/com.vmware.fusion.tools.darwin.zip.tar'))
```
Comment (ProximaB) [inline gettools.py:136]: Seems that it doesn't throw if directory not exist. 😞

**Review by ProximaB — CHANGES_REQUESTED** [review]

File: gettools.py (hunk 1 above)
Comment (ghost) [reply gettools.py:136]: The exe file by pyinstaller cannot throw 404 error.

**Pull request closed without merging**